pkg/engine/lifecycle_test.go

```diff
@@ -3851,3 +3852,27 @@ func TestImport(t *testing.T) {
 		})
 	assert.Nil(t, res)
 }
+
+func TestCustomTimeouts(t *testing.T) {
+	loaders := []*deploytest.ProviderLoader{
+		deploytest.NewProviderLoader("pkgA", semver.MustParse("1.0.0"), func() (plugin.Provider, error) {
+			return &deploytest.Provider{}, nil
+		}),
+	}
+
+	program := deploytest.NewLanguageRuntime(func(_ plugin.RunInfo, monitor *deploytest.ResourceMonitor) error {
+		_, _, _, err := monitor.RegisterResource("pkgA:m:typA", "resA", true, "", false, nil, "",
+			resource.PropertyMap{}, nil, false, "", nil, nil, "", &resource.CustomTimeouts{
+				Create: 60, Delete: 60, Update: 240,
```

Comment:
Can we update this test s.t. the provider for `pkgA` validates that these timeouts are present?

Reply:
done!